training/dist_clm_train.py

```diff
@@ -292,6 +292,7 @@ def main():
     parser.add_argument('--job-id', 
                         type=str, default="0", metavar='S',
                         help='an uuid')
+    parser.add_argument('--epoch-steps', type=int, default=0, help='-')
```

Comment:
We should compute epoch steps. This conflicts with the option that specifies the number of steps.

Reply:
we'd need num-examples which we don't have

Reply:
You will need to get the number of examples from train_data_loader. You may need to add a function to get the number of examples.

Reply:
This probably works.

Reply:
https://github.com/togethercomputer/OpenChatKit/blob/main/training/dist_clm_train.py#L347

Reply:
yup. That's what I used. Fixed in new commit.